README.md

```diff
@@ -109,17 +111,37 @@ specification for a full list of information that can be retrieved.
   ```
  
 ## Get the list of RPMs defining the public API for a particular module NSVC
+First, query the Modulemd.ModuleIndex for the module with a given name.
 ## C
 ```C
 ModulemdModule * module = modulemd_module_index_get_module (merged_index, "modulename");
-ModulemdModuleStream * stream = modulemd_module_get_stream_by_NSVC (module, "modulestream", 1, "deadbeef");
-GStrv api_list = modulemd_module_stream_v2_get_rpm_api_as_strv(stream); 
 ```
 
 ## Python
 ```python
  module = merged_index.get_module ('modulename')
+```
+
+Then, query the Modulemd.Module for the Modulemd.ModuleStream associated with the provided NSVC
+(name-stream-version-context identifier).
+## C
+```C
+ModulemdModuleStream * stream = modulemd_module_get_stream_by_NSVC (module, "modulestream", 1, "deadbeef");
```

Comment:
This needs updating, actually. We recently [replaced](https://github.com/fedora-modularity/libmodulemd/commit/c9708f5f49c28745b3ec7e204d493e66f0ad31ce) this function with `modulemd_module_get_stream_by_NSVCA()`. So we should update these docs not to reference it.